Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM inclusivedesign/nodejs:4.2.1
```

Comment:
Thanks @benjaminstokes - as discussed on IRC, the lack of the 4.2.1 tag was due to a change in how we were building the `inclusivedesign/nodejs` images (we moved from relying on Docker Hub to using an internal Jenkins instance) that happened after the PR was made.

I've updated the Dockerfile to use 4.2.4 and have additionally added jobs to our Jenkins instance to build 4.2.1 and other container tags we want to have available.